validate_sptx/examples/experiment/field_of_view.json

```diff
@@ -0,0 +1,59 @@
+{
+  "version": "0.0.0",
+  "dimensions": [
+      "x",
+      "y",
+      "z",
+      "r",
+      "c"
+  ],
+  "shape": {
+      "z": 45,
+      "r": 8,
+      "c": 2
+  },
+  "default_tile_format": "TIFF",
+  "default_tile_shape": [
+    2048,
+    2048
+  ],
+  "extras": "nothing to see here",
+  "tiles": [
+    {
+      "coordinates": {
+        "xc": [0.0, 0.1],
+        "yc": [0.0, 0.1],
+        "zc": [0.0]
+      },
+      "indices": {
+        "z": 0,
+        "r": 0,
+        "c": 0
+      },
+      "file": "fdaj3902.tiff",
+      "tile_format": "TIFF",
+      "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+      "extras": {
+        "are_we_missing_something":"probably"
+      }
+    },
+    {
+      "coordinates": {
+        "xc": [0.0, 0.1],
```

Comment:
I think our codebase should stop perpetuating the idea that coordinates don't matter :-) Increment either `x` or `y` to start at `0.1`?

Reply:
This is copied from `validate_sptx/examples/field_of_view` but happy to do whatever's appropriate. (Possibly a next PR?)

Reply:
"This is copied". What I _probably_ should do here is to make this a symlink and then I won't be propagating but only maintaining the status quo. (If symlinks are determined copacetic in the repo.)

Reply:
cc @ttung w.r.t. symlinks. 

> This is copied from validate_sptx/examples/field_of_view but happy to do whatever's appropriate. (Possibly a next PR?)

Makes sense, this is a bad habit we (I?) introduced back when we were exclusively dealing with single fields of view. Now that we are more aware of multiple-fov experiments our examples should match expectations. 

Down the road, I could see Starfish throwing a validation error when given two "fields of view" that are in the exact same coordinate space. That seems like it violates our expectations about what a field of view is. 

... slippery slope, since we allow overlapping fovs, so two with 10% offsets should probably be OK. However, I think the example as posed is definitely wrong.

Reply:
I don't think I will be able to get back to this PR within the next. I'd propose landing it (and if I'm offline, please feel free to do that) and I can look into cleaning up the validation tests as a follow-on.

Reply:
🥂 Marking this as resolved, happy to add appropriate validation if desired (e.g. "if all tiles are at the same coordinates, complain loudly")